oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -200,6 +208,12 @@ String createTokenUrlWithScopes() {
     if (!scopes.isEmpty()) {
       tokenUrl.set("scopes", Joiner.on(',').join(scopes));
     }
+    if (!transport.isEmpty()) {
+      tokenUrl.set("transport", transport);
+    }
+    if (!bindingEnforcement.isEmpty()) {
+      tokenUrl.set("binding-enforcement", bindingEnforcement);
+    }
```

Comment:
I think this should be null and empty check. Setter can set a null value unless we want to add a check there

Reply:
Thanks for pointing that out, done in https://github.com/googleapis/google-auth-library-java/pull/1558/commits/9d46bbd880b6240bb7a828510254b075a60c7188